client/client_test.go

```diff
@@ -0,0 +1,98 @@
+package main
+
+import (
+	matmult "github.com/amwolff/2-lg-mats-bench/gen/go"
+	"google.golang.org/grpc"
+	"log"
+	"math"
+	"testing"
+)
+
+func BenchmarkParsePBRequestMessage(b *testing.B) {
+	request, err := parseJSONFile("/tmp/MACIERZE.JSON")
```

Comment:
Let's not hard-code the path like that. Maybe abstract it into some `const` or array of datasets? Allow dataset rotation, make the benchmarks a lot more configurable, versatile and support many input/output formats.

Reply:
That would be a lot better. I'm gonna change it by the end of the week.